index.html

```diff
@@ -202,6 +202,91 @@ <h5 class="modal__headline modal-title" id="ModalLabel">Design.</h5>
                   <!-- /Desctop, large tablet -->
       </div>
     </div>
+    <div class="my-skills container" id="skills">
+      <div class="row">
+        <div class="line-vertical col-12 text-center"></div> <!--pionowa kreska-->
+        <div class="col-12">
+          <h1 class="distinguish text-center mb-4">Umiejętności</h1>
+          <p class="skills-description text-center mx-auto text-white">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi suscipit, repellat id deserunt excepturi labore nostrum amet doloremque provident et.
+          </p>
+        </div>
+        <div class="skills--item col-6 col-sm-4 col-md-3 text-center">
+          <img class="skills--icons" src="images/icons/html-5.svg" alt="html5">
+          <div class="progress">
+            <div class="progress-html progress-bar progress-bar-striped" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"></div>
```

Comment:
tutaj cos nie dziala, animacja nie jest plynna oraz po wypelnieniu progress baru zaczyna sie od poczatku. Mozesz to zrobic statyczne bez animacji :)